routes/services/study/deleteStudy.ts

```
@@ -0,0 +1,11 @@
+import prisma from "../../prisma";
+import { RequestHandler } from "express";
+
+const deleteStudy: RequestHandler = async (req, res, next) => {
+  try {
+  } catch (error) {
+    next(error);
+  }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)